HangFire.Autofac/project.json

```diff
@@ -0,0 +1,17 @@
+{
+	"version": "2.3.0-*",
+	"dependencies": {
+		"NETStandard.Library": "1.6.0",
+		"Hangfire.Core": "1.6.0",
+		"Autofac": "4.0.0"
```

Comment:
Is it breaking change to use `Autofac 4.0.0` instead of `Autofac 3.0.0`? Can we use `Autofac 3.0.0` version for .NET Full Framework and `Autofac 4.0.0` for .NET Core?